### PR TITLE
[OTA-3397] message change follow up : remove the superfluous pac…

### PR DESCRIPTION
**Pull request by piotrcichonmee**
…kage upload modal

Signed-off-by: Piotr Cichon <ext-piotr.cichon@here.com>

### CODE REVIEW

File: ota-plus-web/app/reactapp/src/helpers/timeConverterHelper.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
```
Comment (piotrsmol) [inline ota-plus-web/app/reactapp/src/helpers/timeConverterHelper.js]: .

File: ota-plus-web/app/reactapp/src/partials/OperationCompletedInfo.jsx [hunk 2]
```diff
@@ -33,17 +34,19 @@ class OperationCompletedInfo extends Component {
   }
 
   render() {
-    const { info } = this.props;
+    const { info, notDisappear } = this.props;
```
Comment (piotrsmol) [inline ota-plus-web/app/reactapp/src/partials/OperationCompletedInfo.jsx]: Could you rename `notDisappear` to `preserve` maybe?

File: ota-plus-web/app/reactapp/src/helpers/timeConverterHelper.js (hunk 1 above)
Comment (piotrcichonmee) [reply ota-plus-web/app/reactapp/src/helpers/timeConverterHelper.js]: Done

File: ota-plus-web/app/reactapp/src/partials/OperationCompletedInfo.jsx (hunk 2 above)
Comment (piotrcichonmee) [reply ota-plus-web/app/reactapp/src/partials/OperationCompletedInfo.jsx]: Done

**Pull request merged**